commands/completion.go

```diff
@@ -48,7 +48,10 @@ func init() {
 }
 
 func bashCmdF(cmd *cobra.Command, args []string) {
```

Comment:
Nit: instead of logging the error you can also return it and let cobra command print it
```suggestion
func bashCmdF(cmd *cobra.Command, args []string) error {
```

The you also would have up update https://github.com/mattermost/mmctl/pull/107/files#diff-0f095041bcad19744b6226478e23d155R38 to 
```golang
	RunE: zshCmdF,
```

This is more of an FYI and not blocking for the PR.